### ISSUE TITLE
[SIMULATOR]: Crash on display capture since version 3683

### ISSUE THREAD

**Issue by depilz** [coronalabs/corona#595]
On Mac (Not sure about windows) the simulator suddenly closes when you try to capture an object. It seems to be working fine on device.

Happens on all versions from 3683 onwards.

To reproduce you can use this in your `main.lua`:

```lua
local r = display.newRect(0, 0, 100, 100)
display.capture(r)
```

Strangely, it won't crash a 100% of the times, but it generally does.

**Comment by depilz** [comment]
Also seems to happen only on Apple Silicon macs

**Comment by scottrules44** [comment]
@depilz can you confirm this happen with 2023.3691, I thought pushed a fix for a similar issue a while back.

I also tried refresh a bunch with the code you posted and could not get crash on my m1 machine.

**Comment by depilz** [comment]
Thanks @scottrules44, yes it was indeed fixed on `2023.3691`. Thank you so much.
